examples/sqlite/src/lib.rs

```diff
@@ -1,3 +1,4 @@
+#![recursion_limit = "1024"]
```

Comment:
I wonder where is the recursion? And why increasing the limit convince Rust compiler?

Reply:
For more information about the recursion can be found here https://github.com/async-graphql/async-graphql/issues/1125.

An idea is to generate a GraphQL connection node using derives